test/test_cli.rb

```diff
@@ -1,441 +1,357 @@
 # frozen_string_literal: true
+
 require_relative 'helper'
 require 'sidekiq/cli'
+require 'sidekiq/launcher'
 require 'tempfile'
 
-class Sidekiq::CLI
-  def die(code)
-    @code = code
-  end
-
-  def valid?
-    !@code
-  end
-end
-
-class TestCli < Sidekiq::Test
-  describe 'CLI#parse' do
-
+class TestCLI < Sidekiq::Test
```

Comment:
@mperham Why have you introduced `Sidekiq::Test`? It basically aliases `Minitest::Test`...

Reply:
I don’t recall where that class came from. Agreed, happy to see it removed.